src/config.vala.in

```diff
@@ -0,0 +1,10 @@
+namespace Constants {
```

Comment:
I don't think you need any of this. Where is this intended to be used?

Reply:
The config data is not used, for now, so it's pretty useless indeed. I put it there in case it's necessary to detect and use those constants in Vala. For example, if the version number stored in a gsetting is different from the current version of the app.
If you think this is unusual and unnecessary, I'm good with removing it.
Thanks

Reply:
:man_shrugging: I guess it's not really doing any harm. But also [YAGNI](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it)

Reply:
You're 100% right!